src/transformers/pipelines.py

```diff
@@ -367,14 +367,14 @@ class Pipeline(_ScikitCompat):
     """
 
     default_input_names = None
+    task = None
```

Comment:
@mfuntowicz - To me "task" is a static class variable and should be attached to the class and not given to the `__init__` method. What do you think?